change-is-inevitable-versioning-eds/README.md

```diff
@@ -0,0 +1,7 @@
+# Change is inevitable: versioning event-driven systems
+
+## Abstract
+
+Building an event-driven system is anything but trivial. Still, once you make it past the eventual consistency hurdles, the sea of pub-sub vs. command-response debates, and the service boundaries conundrum, you’ll soon face a new, inevitable adversary: change. The conversations that follow sound all too familiar: “Who’s subscribed to this message?” “Do other services depend on this field in the payload?” “Why on earth is that thing in the payload?” “That service should never rely on this data!” And, of course, the obvious “Can’t we -just- remove this?”
```

Comment:
This first paragraph feels unnecessary when compared to the second paragraph. I would remove it.

Reply:
I think the reason I did this is because I wanted to differentiate, the easy set of questions:
>  “Who’s subscribed to this message?” “Do other services depend on this field in the payload?” “Why on earth is that thing in the payload?” “That service should never rely on this data!” And, of course, the obvious “Can’t we -just- remove this?”

To the necessary questions: 
> How do you measure the impact of tweaking a message contract without forcing other teams beyond their deadlines? Can you make the change without breaking half of the system? Which techniques provide backward compatibility, and, how long should it be maintained? Oh, and let’s not forget that we’re supposed to solve this problem with zero downtime as our users are spread across every time zone

So, I was trying to connect to the first part that may be happening, redirecting to the desired questions to ask. But I think I could condense it.

Reply:
@khalidabuhakmeh I posted a new version without altering the old one as I've asked others to review too 😄 
I reflected on what you said about the first paragraph, but I felt strongly about introducing the "faulty" questions. I did rephrase the second paragraph to be less question-oriented, and more result-oriented, indicating that's what they'll learn. Thanks for sharing your thoughts, this is much appreciated 🙏🏼